edbdeploy/terraform.py

```diff
@@ -100,12 +100,22 @@ def bin(self, binary):
 
     def init(self):
         try:
-            rc = exec_shell_live(
-                [self.bin("terraform"), "init", "-no-color", self.dir],
+            # Added step for Terraform >= 0.15.1
+            rc0 = exec_shell_live(
+                [self.bin("export"), "TF_DATA_DIR=%s" % self.dir],
                 environ=self.environ,
                 cwd=self.dir
             )
```

Comment:
Exporting the `TF_DATA_DIR` env. variable this way (in its own `exec_shell_live()` context) is not the right thing to do. I'd suggest you to make this env. variable available for all `terraform` command by setting it in `TerraformCli.__init__()` like we did for the `TF_PLUGIN_CACHE_DIR` var.:
```python
class TerraformCli:

    def __init__(self, dir, plugin_cache_dir, bin_path=None):
        self.dir = dir
        self.plugin_cache_dir = plugin_cache_dir
        self.environ = os.environ
        self.environ['TF_PLUGIN_CACHE_DIR'] = self.plugin_cache_dir
        self.environ['TF_DATA_DIR'] = self.dir

```